cmake/cubos-config.cmake.in

```diff
@@ -0,0 +1,32 @@
+include(CMakeFindDependencyMacro)
+
+if(NOT @CUBOS_CORE_SHARED@)
+    find_dependency(Threads)
+    find_dependency(cpptrace REQUIRED)
+    
+    if(@CUBOS_CORE_OPENGL@)
+        find_dependency(glad REQUIRED)
+    endif()
+
+    if(@CUBOS_CORE_GLFW@)
+        find_dependency(glfw3 REQUIRED)
+    endif()
+
+    if(@CUBOS_CORE_OPENAL@)
+        find_dependency(OpenAL REQUIRED)
```

Comment:
Might be wrong, but do we still use this? Though that stopped being a dependency after #1339.

Reply:
Oh I see we still have the oal_audio_device, shouldn't we remove this file?

Reply:
Yeah I guess we forgot to remove it in the audio backend PR, oops. Gonna do it here